group_project_v2/mixins.py

```diff
@@ -133,13 +133,28 @@ def _get_setting(self, setting, default):
         return result
 
 
-class AuthXBlockMixin(SettingsMixin, ProjectAPIXBlockMixin):
+class AuthXBlockMixin(SettingsMixin, ProjectAPIXBlockMixin, CourseAwareXBlockMixin):
 
-    DEFAULT_TA_ROLE = ('assistant',)
+    DEFAULT_TA_ROLE = ("assistant", )
 
     _ACCESS_DASHBOARD_ROLE_GROUPS_KEY = "access_dashboard_groups"
     _ACCESS_DASHBOARD_FOR_ALL_ORGS_GROUPS_KEY = "access_dashboard_for_all_orgs_groups"
-    _TA_ROLES_KEY = 'ta_roles'
+    _ACCESS_DASHBOARD_TA_ROLES = "access_dashboard_ta_groups"
+    _TA_ROLES_KEY = "ta_roles"
+
+    @property
+    def see_dashboard_ta_perms(self):
```

Comment:
@e-kolpakov This is tricky, they are both. The name of the ``REST`` endpoint is ``group``, but it is a group of type permission. I figured that in the config files it would be to name them ``GROUP`` so people would know what endpoint they should use). The UPPERCASE variables are named to match their values, but properties use permission term. 

One thing I'd rather not do is change setting values (as these values used are in production).We could leave the values and functions it as it is, or change them all to "groups" or "perms".